privaterelay/templates/faq.html

```diff
@@ -27,14 +27,42 @@ <h2 class="faq-headline">
           What about spam?
         </h2>
         <p class="faq-answer">
-          We use Amazon SES as our Email Service Provider, which enforces industry standard spam controls. We also have the ability to enforce even more strict spam filtering.      </p>
+          While Relay does not filter for spam, our email partner Amazon SES does block spam and malware. If Relay forwards messages you don’t want, you can update your Relay settings to block messages from the alias forwarding them. If you see a broader problem of unwanted email from all of your aliases, please report this to us so we can consider adjusting the SES spam thresholds for this service.
+        </p>
+        <p class="faq-answer">
+          If you report these as spam, your email provider will see Relay as the source of spam, not the original sender.
+        </p>
+      </section>
+      <section class="faq">
+        <h2 class="faq-headline">
+          Can I reply to messages using my Relay alias?
+        </h2>
+        <p class="faq-answer">
+          Relay does not yet offer the ability to reply using an alias. If you try, nothing will happen. We are working on additional features to let you <a href="https://github.com/mozilla/fx-private-relay/issues/151" rel="noopener noreferrer" class="text-link">send an automated response</a> or <a href="https://github.com/mozilla/fx-private-relay/issues/99" rel="noopener noreferrer" class="text-link">reply anonymously to senders</a>.
```

Comment:
Hmm ... https://github.com/mozilla/fx-private-relay/issues/151 is not for the *users* to send an automated response. It's for *us* to send an automated response reminding users that they can't yet reply to relay emails.

Reply:
drop the link?

Reply:
Yeah, I'd drop the link for sure. Could drop that mention completely too? If you think Michelle would be fine with that.